docs/run.md

```diff
@@ -12,9 +12,15 @@ go run github.com/onosproject/onos-config/cmd/onos-config \
     -deviceStore=$HOME/go/src/github.com/onosproject/onos-config/configs/deviceStore-sample.json \
     -networkStore=$HOME/go/src/github.com/onosproject/onos-config/configs/networkStore-sample.json \
     -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/TestDevice-1.0.0/testdevice.so.1.0.0 \
-    -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/TestDevice-2.0.0/testdevice.so.1.0.0 \
+    -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/TestDevice-2.0.0/testdevice.so.2.0.0 \
     -modelPlugin=$HOME/go/src/github.com/onosproject/onos-config/modelplugin/Devicesim-1.0.0/devicesim.so.1.0.0
 ```
+> This gives a list of model plugins to load at startup. Alternatively these can
+> be omitted here and loaded later with the **onos load-plugin** command. The
+> plugins here were built locally with a command like
+> **go build -o modelplugin/TestDevice-1.0.0/testdevice.so.1.0.0 -buildmode=plugin ./modelplugin/TestDevice-1.0.0**
```

Comment:
I'd suggest that for commands we use the back-tick command mark-down instead.

Reply:
Done